apps/aecontract/src/aect_evm.erl

```diff
@@ -105,7 +105,7 @@ execute_call(#{ code := CodeAsHexBinString
            pre => #{}},
     TraceSpec =
         #{ trace_fun =>
-               fun(S,A) -> io_lib:format(S,A) end
+               fun(S,A) -> io:format(S,A) end
```

Comment:
Was this intentional?

Reply:
Yes. The `trace_fun` is used as `io:format/2` rather than `io_lib:format/2` (see `aevm_eeevm_state:trace_format/3`), so using `io_lib:format/2` is equivalent to `fun(_, _) -> ok end`.